semgrep/tests/e2e/snapshots/test_rule_parser/test_rule_parser__failure__error_messages/badpattern/error.json

```diff
@@ -3,14 +3,14 @@
     {
       "code": 2,
       "level": "error",
-      "message": "[ERROR] Semgrep Core \u2014 Pattern parse error\nAn error occurred while invoking the Semgrep engine. Please help us fix this by creating an issue at https://github.com/returntocorp/semgrep\n\nIn rule eqeq-is-bad: Invalid pattern for Python\n",
-      "rule_id": "eqeq-is-bad",
+      "message": "[ERROR] Semgrep Core \u2014 Pattern parse error\nAn error occurred while invoking the Semgrep engine. Please help us fix this by creating an issue at https://github.com/returntocorp/semgrep\n\nIn rule rules.syntax.eqeq-is-bad: Invalid pattern for Python\n",
+      "rule_id": "rules.syntax.eqeq-is-bad",
       "spans": [
         {
           "config_end": {
             "col": 24,
             "line": 4,
-            "offset": 60
+            "offset": 73
```

Comment:
Are the offset changes here due to rewriting the rule yaml file?

Reply:
Yes, `rules.syntax.` is 13 characters

Reply:
Yes, `rules.syntax.` is 13 characters